### ISSUE TITLE
Document the Owkin Team algorithm

### ISSUE THREAD

**Issue by reubano** [concept-to-clinic/concept-to-clinic#28]
## Overview

Participants in the [Data Science Bowl](https://www.kaggle.com/c/data-science-bowl-2017) produced several algorithms that we would like to incorporate. To help facilitate this effort, we also want to add documentation so that contributors can make an educated decision when selecting an algorithm to incorporate.

## Expected Behavior

This documentation should enable people to:
 - view the library dependencies and license
 - understand its pros/cons
 - evaluate its performance/accuracy
 - identify which areas of the codebase to target for improvement

Design doc reference: [Detect and select](https://concept-to-clinic.readthedocs.io/en/latest/design-doc.html#detect-and-select)

## Algorithm info

key | value
----|------
team|Owkin Team
rank|10
repo|https://github.com/owkin/DSB2017
trained models|https://github.com/owkin/DSB2017/blob/master/sje_scripts/Unet_X.hdf5
trained models|https://github.com/owkin/DSB2017/blob/master/sje_scripts/Unet_Y.hdf5
trained models|https://github.com/owkin/DSB2017/blob/master/sje_scripts/Unet_Z.hdf5
trained models|https://github.com/owkin/DSB2017/blob/master/pic_scripts/model64x64x64_v5_rotate_v2.h5
trained models|xgboost trees/features not provided
converted branch|
ML engine|Keras
engine-version|2
ML backend|Tensorflow
backend-version|1
training method|
architecture|
algorithm|
OS|
OS version|
Python version|
CUDA version|
cuDNN version|
notes|http://pyradiomics.readthedocs.io/en/latest/installation.html

## Technical details

 - This feature should be implemented as a new markdown file in the [`docs folder`](https://github.com/concept-to-clinic/concept-to-clinic/tree/master/docs)

## Acceptance criteria

 - [ ] effective documentation for the above

*NOTE: All PRs must follow the standard PR checklist.*
